src/sentry/migrations/0729_backfill_groupsearchviews_with_pinned_searches.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 5.0.6 on 2024-06-03 17:27
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from sentry.models.savedsearch import SortOptions
```

Comment:
I recommend just copy/pasting this into the migration. We usually prefer to avoid importing external code if we can avoid it, so that the migration remains self contained.